browser/ui/webui/brave_shields/shields_panel_data_handler.cc

```diff
@@ -22,21 +22,20 @@ ShieldsPanelDataHandler::ShieldsPanelDataHandler(
   DCHECK(profile);
   Browser* browser = chrome::FindLastActiveWithProfile(profile);
   browser->tab_strip_model()->AddObserver(this);
-  auto* shields_data_ctrlr = GetActiveShieldsDataController();
-  if (!shields_data_ctrlr)
+  observed_shields_data_ctrl_ = GetActiveShieldsDataController();
```

Comment:
nit: `observed` here is unnecessary and slightly confusing imo. I would call it `active_shields_data_controller` and definitely drop the ctrl in favor of controller.